crates/egui/src/load.rs

```diff
@@ -96,6 +99,19 @@ pub enum LoadError {
     Loading(String),
 }
 
+impl LoadError {
+    pub fn size(&self) -> usize {
```

Comment:
let's give it a more descriptive name:

```suggestion
    pub fn byte_size(&self) -> usize {
```